AirCasting/CoreData/Storages/MeasurementStreamStorage.swift

```diff
@@ -289,6 +289,15 @@ final class HiddenCoreDataMeasurementStreamStorage: MeasurementStreamStorageCont
         }
     }
     
+    func checkForLocationlessSessions() throws -> Bool {
+        do {
+            return try context.checkForLocationlessSessions()
+        } catch {
+            Log.error("Error when checking for locationless sessions: \(error.localizedDescription)")
+        }
+        return false
+    }
```

Comment:
Here maybe it would be cleaner to use return false inside catch as it should be the only reason when we should use literal `false` I think